### DISCUSSION TITLE
how do i get a specific row?

### DISCUSSION THREAD

**Discussion by tryggs** [supabase/supabase#7544] [Questions] [answered]
lets say i have a table with a bunch off people with a bunch of different data about them.

how do I get one specific based on its primary id?

**Comment by steve-chavez** [comment, marked as answer]
Check https://supabase.com/docs/reference/javascript/eq

**Reply by Shivansh-yadav13** [reply]
@steve-chavez How can I do it in python??

**Reply by steve-chavez** [reply]
See https://github.com/supabase-community/postgrest-py/#read
